src/DataSource.ts

```diff
@@ -291,6 +291,9 @@ export class DataSource extends DataSourceWithBackend<TimestreamQuery, Timestrea
       range,
     } as unknown) as DataQueryRequest).pipe(
       map((res) => {
+        if (res.error) {
+          throw new Error(res.error.message);
```

Comment:
would it be possible for `res.error` to not have a message? If so, should we handle that case with `res.error?.message`?

Reply:
oops, I meant to check if there is a message, not that we want to throw an error with no message 😄

Reply:
by definition yes, message is optional, but in practice don't think so